apostrophecms/svg-sprite
# PRO-2384 Beta

## Summary

Initial port of the apostrophe-svg-sprites module for A3.

## What are the specific steps to test this change?

1. Use the `svg-sprite` branch of the internal testbed project. That includes an SVG file to import.
2. Run the import task: `node app @apostrophecms/svg-sprite:import`
3. Check the SVG Sprite pieces in their manager. Also check the db for the `file` property, which isn't in the UI.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
